openbb_terminal/forecast/theta_model.py

```diff
@@ -96,12 +96,19 @@ def get_theta_data(
             season_mode=seasonal,
             seasonality_period=seasonal_periods,
         )
-        model.fit(train)
-        pred_theta = model.predict(len(val))
-        res = mape(val, pred_theta)
-        if res < best_mape:
-            best_mape = res
-            best_theta = theta
+        try:
+            model.fit(train)
+            pred_theta = model.predict(len(val))
+            res = mape(val, pred_theta)
+            if res < best_mape:
+                best_mape = res
+                best_theta = theta
+        except Exception:  # noqa
+            continue
+
+    if best_theta == 0:
+        console.print("Theta Model failed to find best theta")
```

Comment:
Do you think we could tell the user why? In the previous it mentioned lack of data.

Reply:
If you don't mind, I'd like to add 1 quick sentence for that such as: 
"Please consider using a series with at least 50 elements."